### PR TITLE
README: remove Maintainers

### PR DESCRIPTION
**Pull request by jonboulle**
Redundant with the MAINTAINERS file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,16 +54,6 @@ A: Existing formats can continue to be a proving ground for technologies, as nee
 * June ??? v1.0.0
   * Release initial version of spec with two independent non-experimental implementations from OCI members
 
-## Maintainers
```
Comment (vbatts) [inline README.md]: Perhaps a note to 'see the MAINTAINERS file' or similar

Comment (jonboulle) [inline README.md]: I considered that but this README is already too long, and I think it's discoverable enough from the main repo landing page...

Comment (vbatts) [inline README.md]: fair. While copying over the contributing and git commit style, I was thinking the README's should get tidied up.

Comment (jonboulle) [inline README.md]: :baby: steps

**Pull request merged**